itou/users/migrations/0043_prevent_pe_connect_duplicates.py

```diff
@@ -0,0 +1,38 @@
+import time
+
+from django.db import IntegrityError, migrations, transaction
+
+
+def fill_username_from_allauth(apps, schema_editor):
+    SocialAccount = apps.get_model("socialaccount", "SocialAccount")
+    total = updated = 0
+    for account in SocialAccount.objects.select_related("user").filter(
+        provider="peamu",  # all social accounts come from peamu, but just in case.
+    ):
+        total += 1
+        if total % 100 == 0:
+            # Go easy on the DB.
+            time.sleep(2)
```

Comment:
```suggestion
        if updated % 1000 == 0:
            # Go easy on the DB.
            time.sleep(1)
```
`total` -> `updated` : Vu que c'est les surtout les écritures qui vont embeter la DB, même si la différence entre les deux variables doit pas être énorme de ce que j'ai compris.
`% 1000` et `sleep(1)` : La DB à vue bien pire et n'a pas spécialement bronché avec des valeurs plus agressive donc autant que le script dure le moins longtemps possible.